hack/build-images.sh

```diff
@@ -182,7 +182,7 @@ function login_to_registry() {
       username=kubeadmin
     fi
   fi
-  podman login --tls-verify=false -u "$username" -p "$token" "$1" > /dev/null
+  docker login -u "$username" -p "$token" "$1" > /dev/null
```

Comment:
why not? I've been using this (even today)

Reply:
What version of podman, docker, and oc are you using?

Reply:
see https://github.com/openshift/cluster-logging-operator/pull/253

Reply:
.... I built my own. So that might not be very helpful info. Lets disregard my comment in the meantime in that case.

Reply:
So... I was following other steps, not necessarily using this script. imagebuilder seems to require docker anyway.